src/main/java/frc/robot/constants/GameRobotConstants.java

```diff
@@ -96,6 +113,19 @@ public int getAimerMotorIdRight() {
         return 22;
     }
 
+    @Override
+    public PID getAimerPid() {
+        return new PID(0.4, 0, 0);
```

Comment:
A lot of these feel like subsystem-specific constants that depend only on the design of the aimer, and could live inside the Aimer class as plain double constants.

Reply:
Agreed. We had decided to put them in Constants so that when tuning the robot one only needs to make updates in Constants. We can reevaluate that approach.